examples/all-clusters-app/esp32/README.md

```diff
@@ -77,13 +71,9 @@ make sure the IDF_PATH has been exported(See the manual setup steps above).
         `Component config`->`CHIP Device Layer`->`WiFi Station Options`->`Default WiFi SSID` and
         `Default WiFi Password` respectively.
 
-        To use the default configuration options, run the default config
-
-          $ idf make defconfig
+-   To build the demo application.
 
--   Run make to build the demo application.
-
-          $ idf make
+          $ idf.py build
```

Comment:
Does that automatically select reasonable `-j` options or equivalent?

That is, how do I do the equivalent of `idf make -j16 flash monitor ESPPORT=/dev/tty.SLAB_USBtoUART` in this new setup?

Reply:
@bzbarsky-apple I think, idf.py should internally take care of parallel builds. I will confirm this internally and get back.

Reply:
@bzbarsky-apple Yes, confirmed, idf.py handles it internally. You need not add anything else